massa-models/src/address.rs

```diff
@@ -114,9 +136,7 @@ impl Address {
     /// assert_eq!(address, res_addr);
     /// ```
     pub fn from_bytes(data: &[u8; ADDRESS_SIZE_BYTES]) -> Result<Address, ModelsError> {
```

Comment:
The signature of `Hash::from_bytes` doesn't return errors anymore so there is no case where this function return error.  (and same for all ids in the other files)